### PR TITLE
Fix failing 3d-tiles test

### PR DESCRIPTION
**Pull request by lilleyse**
Just waiting until the CI passes.

### CODE REVIEW

**Comment by pjcozzi** [comment]
There's failures in release: https://travis-ci.org/AnalyticalGraphicsInc/cesium/builds/210688218

**Comment by lilleyse** [comment]
There were two intermittent failures which I can't reproduce locally with the same command that travis runs.

I adjusted the 404-related test to give it a greater timeout and changed the file name, which seems to have helped. I haven't seen the other test about `ModelInstanceCollection` culling in a while, so maybe it just works now...

The jsHint errors appeared recently and are also fixed.

File: Specs/Scene/Cesium3DTilesetSpec.js [hunk 1]
```diff
@@ -172,7 +172,7 @@ defineSuite([
 
     it('rejects readyPromise with invalid tileset.json', function() {
         var tileset = scene.primitives.add(new Cesium3DTileset({
-            url : 'invalid'
+            url : 'invalid.json'
```
Comment (mramato) [inline Specs/Scene/Cesium3DTilesetSpec.js]: This test should mock the loadJson function instead or relying on the server running the tests to actually fail.

Comment (lilleyse) [inline Specs/Scene/Cesium3DTilesetSpec.js]: After sweeping over the code this seems to be the common practice. Just as an example, search for `fail('should not resolve');`

Is it worth opening an issue for this?

Comment (mramato) [inline Specs/Scene/Cesium3DTilesetSpec.js]: Nah, that's OK.  Nowhere near a priority for us.  If it ever comes up again we can worry about it then.

**Pull request merged**